catalog/ngis-cat/projects_ngis-cat_assets_DEA_NIDEM.jsonnet

```diff
@@ -0,0 +1,165 @@
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+local catalog_subdir_url = ee_const.catalog_base + subdir + '/';
+local parent_url = catalog_subdir_url + 'catalog.json';
+local self_url = catalog_subdir_url + base_filename;
+
+{
+  // TODO(schwehr): Remove when the dataset is ready.
+  'gee:skip_indexing': true,
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  version: version,
+  title: 'National Intertidal Digital Elevation Model 25m ' + version,
+  'gee:type': ee_const.gee_type.image,
+  description: |||
+    The National Intertidal Digital Elevation Model (NIDEM; Bishop-Taylor et
+    al. 2018, 2019) is a continental-scale elevation dataset for Australia's
+    exposed intertidal zone.
+    NIDEM provides the first three-dimensional representation of Australia's
+    intertidal sandy beaches and shores, tidal flats and rocky shores and reefs
+    at 25 m spatial resolution, addressing a key gap between the availability
+    of sub-tidal bathymetry and terrestrial elevation data. NIDEM was generated
+    by combining global tidal modelling with a 30-year time series archive of
+    spatially and spectrally calibrated Landsat satellite data managed within
+    the Digital Earth Australia (DEA) platform.
+    NIDEM complements existing intertidal extent products, and provides data to
+    support a new suite of use cases that require a more detailed understanding
+    of the three-dimensional topography of the intertidal zone, such as
+    hydrodynamic modelling, coastal risk management and ecological habitat
+    mapping.
+    For more information, please see the
+    [DEA Intertidal Elevation](https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#basics)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+      ee.link.license(license.reference),
+      {
+        rel: ee_const.rel.cite_as,
+        href: 'https://doi.org/10.1016/j.ecss.2019.03.006',
+        type: ee_const.media_type.html,
+      },
+      {
+        rel: ee_const.rel.source,
+        href: 'https://dapds00.nci.org.au/thredds/catalog/fk4/datacube/002/NIDEM/geotiff/catalog.html',
+      },
+    ],
+  keywords: [
+    'australia',
+    // 'dea_australia',
+    'dem',
+    // 'nidem',
+  ],
+  providers: [
+    ee.producer_provider(
+      'Digital Earth Australia',
+      'https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat'),
+    ee.processor_provider('NGIS', 'https://ngis.com.au/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(
+    108.81, -44.41, 157.82, -9.13,
+    '1986-08-16T00:00:00Z', '2017-07-31T23:59:59Z'),
+  summaries: {
+    gsd: [25],
+    'eo:bands': [
+      {
+        name: 'nidem',
+        description: |||
+          Primary product: elevation in meter units relative to modelled Mean
+          Sea Level.
```

Comment:
`Sea Level` → `Mean Sea Level (MSL)`

Assuming this isn't using one of the other sea levels like MLLW, MLW, MHW, MHHW, ...

Looking below, it looks like this is indeed MSL.

Reply:
The description shown here is "Mean Sea Level". I've added the acronym after it.